automatic/paint.net/tools/chocolateyInstall.ps1

```diff
@@ -1,15 +1,13 @@
-﻿try {
-  $toolsDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
-  Install-ChocolateyZipPackage 'paint.net' '{{DownloadUrl}}' $toolsDir
-
-  $paintFileFullPath = get-childitem $toolsDir -recurse -include *.exe | select -First 1
-  Install-ChocolateyInstallPackage 'paint.net' 'exe' '/auto DESKTOPSHORTCUT=0' "$paintFileFullPath"
-
-  Remove-Item "$paintFileFullPath"
-
-  Write-ChocolateySuccess 'paint.net'
-} catch {
-  Write-ChocolateyFailure 'paint.net' "$($_.Exception.Message)"
-  throw
-}
-
+﻿$packageName = 'paint.net'
+$fileType = 'EXE'
+$silentArgs = '/auto DESKTOPSHORTCUT=0'
+$url = '{{DownloadUrl}}'
+
+# Zipped installer
+Get-ChocolateyWebFile $packageName "$ENV:Temp\paint.net.install.zip" $url
+Get-ChocolateyUnzip "$ENV:Temp\paint.net.install.zip" "$ENV:Temp\pdn"
+$installFile = Get-ChildItem -Path "$ENV:Temp\pdn" -Recurse -Include "*.exe" | Select-Object -First 1
+
+# elevation bug
+#Install-ChocolateyPackage $packageName $fileType $silentArgs $installFile
+Start-Process -FilePath "$installFile" -ArgumentList "$silentArgs" -Verb Runas

```

Comment:
Unfortunately there is so much more transparent to the package things that can happen with the calls through to install-chocolateyinstallpackage related to getting the user input variables and whether or not they want it to be silent. This is not going to work.